native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/dsl/GraalVMExtension.java

```diff
@@ -64,6 +67,14 @@ public interface GraalVMExtension {
      */
     Property<Boolean> getTestSupport();
 
+    @Nested
+    AgentOptions getAgent();
+
+    @Nested
+    CopyMetadataOptions getCopyMetadata();
```

Comment:
this should probably live under `AgentOptions`

Reply:
I agree, I've moved this to `AgentOptions` and renamed it to `metadataCopy` to remain consistent with the naming of other tasks